shared/src/main/scala/com/nitro/nmesos/config/model.scala

```diff
@@ -40,7 +40,8 @@ object model {
 
   case class PortMap(
     containerPort: Int,
-    hostPort: Option[Int]
+    hostPort: Option[Int],
+    protocols: Seq[String]
```

Comment:
I could leave this as a String and simplify the parser code a bit, since we're sending to Singularity the concatenated list of protocols anyway. Let me know if you'd prefer that.

Reply:
no strong opinion here, but `Option[String]` sounds better in this case.